src/index.html

```diff
@@ -11,8 +11,60 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Stylesheets with a .scss extension won't work directly in browsers. You need to compile them to .css files. Change the href attribute to point to a .css file.